ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -847,6 +847,26 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-multi-vcenter-tp-f28
```

Comment:
We do not run y-stream upgrade jobs for tp jobs.

Reply:
oh, so i need upgrade from 4.18 stable to nightly right?

Reply:
will close this pr as it already messed. 
will update in pr:https://github.com/openshift/release/pull/59500

Reply:
cc @duanwei33